src/power_grid_model/validation/validation.py

```diff
@@ -221,6 +221,43 @@ def validate_ids_exist(update_data: Dict[str, np.ndarray], input_data: SingleDat
     return list(chain(*errors))
 
 
+def process_power_sigma_and_p_q_sigma(
```

Comment:
since it actually has some behaviour that a user should never directly call, please make this function hidden from the module interface

```suggestion
def _process_power_sigma_and_p_q_sigma(
```

Reply:
this function should be module-internal. no user should ever want to do this unless they really know what they are doing (and even then, they are most likely better off writing their own function because we don't guarantee a stable API for functions prefixed `_` by PEP convention)

Reply:
Included in the latest commit.